### ISSUE TITLE
Alt text for image resources

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ocw-hugo-themes#260]
As a screen-reader user, or a low-bandwith user, I need alt text on all images. 

Alt text is a metadata field on image resources. 

#### Acceptance Criteria:

- [ ] on the course home page (CHP) the course image needs alt text
- [ ] any embedded images need alt text
- [ ] the resource page template for an image needs alt text (assuming the image is embedded in it)

#### Out of Scope

- Staging a course without alt text should give a warning
- Publishing a course without alt text should fail
